Svc/PassiveRateGroup/docs/sdd.md

```diff
@@ -0,0 +1,82 @@
+<title>PassiveRateGroup Component SDD</title>
+# RateGroupDriver Component
+
+## 1. Introduction
+
+`Svc::PassiveRateGroup` is an passive component that drives a set of components connected to `Svc::Sched` output ports. It contains an synchronous input `Svc::Cycle` port that drives all the operations.  The component invokes each output port in order, passing an argument indicating the order. It tracks execution time of the cycle.
```

Comment:
Maybe this was originally my poor writing, but change the second to last sentence to say "passing an argument specified in the supplied context list."